src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -131,12 +132,38 @@ protected function isPropertyMapped(string $property, string $resourceClass, boo
      * Determines whether the given property is nested.
      *
      * @param string $property
+     * @param string $resourceClass
+     *
+     * @return bool
+     */
+    protected function isPropertyNested(string $property, string $resourceClass): bool
+    {
+        if (false === $pos = strpos($property, '.')) {
+            return false;
+        }
+
+        $metadata = $this->getClassMetadata($resourceClass);
+
+        return $metadata->hasAssociation(substr($property, 0, $pos));
+    }
+
+    /**
+     * Determines whether the given property is embedded.
+     *
+     * @param string $property
+     * @param string $resourceClass
      *
      * @return bool
      */
-    protected function isPropertyNested(string $property): bool
+    protected function isPropertyEmbedded(string $property, string $resourceClass): bool
     {
-        return false !== strpos($property, '.');
+        if (false === $pos = strrpos($property, '.', -1)) {
```

Comment:
@meyerbaptiste , thanks for reviews. @teohhanhui already proposed me a code changes. What is the right way to do it ? Just to prevent a minimal change, which way I have to take when I propose this kind of code ?